puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/source/hammer-cli-x.groovy

```diff
@@ -68,9 +68,11 @@ pipeline {
 }
 
 def add_hammer_cli_git_repos(repos = []) {
+    content = ''
```

Comment:
content *could* be an array and we *could* join it with `"\n"`, but doesn't have to